jit/v2/JIT_v2.0.0.yaml

```diff
@@ -1382,6 +1921,28 @@ components:
           description: |
             A sequential number assigned to each **Terminal Call** within a **Port Call**, indicating the order in which the calls are scheduled to occur. This number helps differentiate multiple terminal visits in the same **Port Call**. 
           example: 2
+        carrierServiceName:
+          type: string
+          maxLength: 50
+          pattern: ^\S(?:.*\S)?$
+          description: |
+            The carrier specific name of the service for which the schedule details are published
+          example: Great Lion Service
+        carrierServiceCode:
+          type: string
+          maxLength: 11
+          pattern: ^\S(?:.*\S)?$
+          description: |
+            The carrier specific code of the service for which the schedule details are published.
+          example: FE1
+        universalServiceReference:
+          type: string
+          pattern: ^SR\d{5}[A-Z]$
+          minLength: 8
+          maxLength: 8
+          description: |
+            A global unique (Vessel-)service reference, as per DCSA standard, agreed by VSA partners for the (Vessel-)service. The (Vessel-)service reference must match the regular expression pattern: `SR\d{5}[A-Z]`. The letters `SR` followed by `5 digits`, followed by a checksum-character as a capital letter from `A to Z`.
```

Comment:
As per Emilio the USR description should be - "A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. The service reference must match the regular expression pattern: SR\d{5}[A-Z]. The letters SR followed by 5 digits, followed by a checksum-character as a capital letter from A to Z."

Reply:
@emiliodcsa I guess we still need to distinguish between Services that are for Vessels (which is the case here) and Services in the context of JIT-Services...? Should I change the wording?

Reply:
@emiliodcsa says only these three values in this version (any updates will require a new major release...)

Reply:
We will keep as it is to make sure it is a vessel service...